recipe/meta.yaml

```diff
@@ -32,7 +37,6 @@ requirements:
   run:
     - python
     - libffi >={{ libffi }}
```

Comment:
is this really necessary?  should we simply use libffi as dependency here?

Reply:
You are right, the plain `libffi` will be better. I've fixed it